Dockerfile

```diff
@@ -1,6 +1,8 @@
-FROM node:14-alpine
-WORKDIR /usr/src/app
-COPY package.json ./
-RUN npm install
-COPY ./src ./src
-CMD npm start
\ No newline at end of file
+FROM rust:1.68.1 AS builder
+RUN apt-get update && apt-get -y upgrade
+COPY . .
+RUN cargo build --release
+
+FROM debian:buster-slim
+COPY --from=builder ./target/release/blob-indexer ./target/release/blob-indexer
+CMD ["/target/release/blob-indexer"]
```

Comment:
I am admittedly not super well-versed in rust-based CI, but Rust/Cargo, due to image size & build time concerns tends to have a very idiosyncratic way of handing CIs. [Here's a post](https://www.lpalmieri.com/posts/2020-11-01-zero-to-production-5-how-to-deploy-a-rust-application/) by Luca Palmieri that goes over it; it's a bit (too) long, but the important thing is the dockerfile itself; we might wanna give it an eye before merging.

Reply:
Thanks for sharing. I'll review it and made the relevant changes before we merge!